packages/samples/headless-react/src/components/relevance-inspector/relevance-inspector.class.tsx

```diff
@@ -52,6 +52,20 @@ export class RelevanceInspector extends Component<{}, RelevanceInspectorState> {
             }
           />
         </label>
+        <label>
```

Comment:
Can we leverage that method in our Atomic component too?

Reply:
I've looked at it, and for now I'd say no.

Implementing a full fledge atomic debug panel/view is going to be part of a different PR/different story. It's more than just tacking on couple buttons.